packages/wrangler/src/index.ts

```diff
@@ -290,103 +308,208 @@ export function createCLIParser(argv: string[]) {
 		}
 	);
 
-	// You will note that we use the form for all commands where we use the builder function
-	// to define options and subcommands.
-	// Further we return the result of this builder even though it's not completely necessary.
-	// The reason is that it's required for type inference of the args in the handle function.
-	// I wish we could enforce this pattern, but this comment will have to do for now.
-	// (It's also annoying that choices[] doesn't get inferred as an enum. 🤷‍♂.)
-
+	/*
+	 * You will note that we use the form for all commands where we use the builder function
+	 * to define options and subcommands.
+	 * Further we return the result of this builder even though it's not completely necessary.
+	 * The reason is that it's required for type inference of the args in the handle function.
+	 * I wish we could enforce this pattern, but this comment will have to do for now.
+	 * (It's also annoying that choices[] doesn't get inferred as an enum. 🤷‍♂.)
+	 */
+	/*
+	 * TODO: Implement proper command grouping if yargs will ever support it
+	 * (see https://github.com/yargs/yargs/issues/684)
+	 * Until then, use a new line in the command description whenever we want
+	 * to create some logical spacing between commands. This is hacky but
+	 * ¯\_(ツ)_/¯
+	 */
+	/******************************************************/
+	/*                 WRANGLER COMMANDS                  */
+	/******************************************************/
 	// docs
 	wrangler.command(
-		"docs [command..]",
-		"📚 Open wrangler's docs in your browser",
+		"docs [command]",
+		"📚 Open the wrangler commands documentation in your browser\n",
 		docsOptions,
 		docsHandler
 	);
 
+	/******************** CMD GROUP ***********************/
 	// init
 	wrangler.command(
 		"init [name]",
-		"📥 Initialize a basic Worker project, including a wrangler.toml file",
+		"📥 Initialize a basic Worker application",
 		initOptions,
 		initHandler
 	);
 
-	// generate
-	wrangler.command(
-		"generate [name] [template]",
-		"✨ Generate a new Worker project from an existing Worker template. See https://github.com/cloudflare/workers-sdk/tree/main/templates",
-		generateOptions,
-		generateHandler
-	);
-
-	// [DEPRECATED] build
-	wrangler.command("build", false, buildOptions, buildHandler);
-
-	// [DEPRECATED] config
-	wrangler.command("config", false, noOpOptions, configHandler);
-
 	// dev
 	wrangler.command(
 		"dev [script]",
-		"👂 Start a local server for developing your worker",
+		"👂 Start a local server for developing a Worker",
```

Comment:
Why this change? It's not just `a` Worker, it's specifically `your` worker in the current directory

Reply:
no idea. The wording came from design, but I'll revert back to `your`